src/lib/utils/query-builder.ts

```diff
@@ -0,0 +1,16 @@
+export const QUERY_PARAM_NAME = {
+    FILTER: "filter",
+}
+
+export const FILTER_PARAM = {
+    TITLE: "title",
+}
```

Comment:
Do these two constants have to be exported if they're only used in this file?

Usually we'd export something that we use in other places as well. I feel like it doesn't apply for these constants, so I'd suggest removing the `export` part.

Reply:
They are used in `__test__/utils/query-builder.test.ts` therefore I made exports on them

Reply:
I completely forgot about the tests, so now it makes sense.

I'm just wondering if there's a way to mitigate this that is not complicated.

The reason is that we don't want to make these constants available to users. They don't need these constants, we just use them to build queries. But to address this there may be a chance of overcomplicating the code and even get to code duplication, which is something we prioritise slightly more in this case.

Let me know if there's a decent way of passing these to the tests without the need of exporting. If not, we can leave the code as is.

Reply:
I see the idea that better to expose from the module only methods that we want to be used.

We can remove exports from the constants and provide in the tests exact string values. I think that will be better in terms on the test to be independent. The only downside is if we change some value in the constraints we will need to make it accordingly in the test. It's not a big deal I believe.

Let me know what you think.

Reply:
We'll leave the exports since we want to reduce code duplication as much as possible.